pkgs/by-name/en/envision-unwrapped/package.nix

```diff
@@ -31,23 +31,19 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "envision-unwrapped";
-  version = "0-unstable-2024-09-28";
+  version = "0-unstable-2024-10-20";
 
   src = fetchFromGitLab {
     owner = "gabmus";
     repo = "envision";
-    rev = "56d500a9f914ce2ddad038223711192e4d1dcbe1";
-    hash = "sha256-8wU2sjhH026l6a11XZ5Qdu5x/EbI+ZqwE7AixsYMCFk=";
+    rev = "c40a4ad05a8e6ea99eed4a7d7d2098a08686e065";
+    hash = "sha256-C/m5Hx52fFyuVI87EmHpe5YqjwDWoyveiXA0sJTt2NQ=";
   };
 
   strictDeps = true;
 
   cargoDeps = rustPlatform.importCargoLock {
     lockFile = ./Cargo.lock;
-    outputHashes = {
-      "libmonado-rs-0.1.0" = "sha256-xztevBUaYBm5G3A0ZTb+3GV3g1IAU3SzfSS5BBqfp1Y=";
-      "openxr-0.18.0" = "sha256-ktkbhmExstkNJDYM/HYOwAwv3acex7P9SP0KMAOKhQk=";
-    };
   };
```

Comment:
I think we should use `cargoHash` instead as we don't need to track git dependencies anymore

Reply:
Good call.

Reply:
You need `rustPlatform.fetchCargoTarball` here ^^